### PR TITLE
Use of KOTEST_VERSION property

### PR DESCRIPTION
**Pull request by rachelcarmena**
In order to help to the progressive use of Kotest.

**Note**: it corresponds to `4.1.0` instead of `4.0.6`. Let's see if it works...

### CODE REVIEW

File: arrow-fx-coroutines/src/test/kotlin/arrow/fx/coroutines/predef-test.kt [hunk 1]
```diff
@@ -111,13 +110,9 @@ fun <L, R> Arb.Companion.either(left: Arb<L>, right: Arb<R>): Arb<Either<L, R>>
...
 }
 
-fun Arb.Companion.intRange(min: Int = 0, max: Int = 1000): Arb<IntRange> =
-  Arb.int(min, max).flatMap { a ->
-    Arb.int(min, max).map { b ->
-      val first = min(a, b)
-      val last = max(a, b)
-      first..last
-    }
+fun Arb.Companion.intRange(min: Int = Int.MIN_VALUE, max: Int = Int.MAX_VALUE): Arb<IntRange> =
```
Comment (rachelcarmena) [inline arrow-fx-coroutines/src/test/kotlin/arrow/fx/coroutines/predef-test.kt:113]: @nomisRev , thank you so much for making it work for Kotest `4.1.0` :tada: Just a small question, could this change impact on current tests? I mean, the max value changed from 1000 to 2147483647. Thanks in advance!!

Comment (rachelcarmena) [reply arrow-fx-coroutines/src/test/kotlin/arrow/fx/coroutines/predef-test.kt:113]: And the min value also changes (from 0 to -2147483648).

Comment (nomisRev) [reply arrow-fx-coroutines/src/test/kotlin/arrow/fx/coroutines/predef-test.kt:113]: Hey @rachelcarmena,
The reason for this change was to make the generator more accurate.
The only current usage from this `Arb` explicitly defines `min` & `max`, so it doesn't impact the current suite.

Comment (rachelcarmena) [reply arrow-fx-coroutines/src/test/kotlin/arrow/fx/coroutines/predef-test.kt:113]: Great, thank you so much Simon!! :raised_hands:

**Pull request merged**